jenkins/getPlatform.py

```diff
@@ -24,7 +24,7 @@
    elif re.search('Fedora', dist[0]):
       osvers = 'fedora' + dist[1].split('.')[0]
    else:
-      osvers = 'linux' + string.join(platform.linux_distribution()[1].split('.')[:2],'')
+      osvers = 'linux'
```

Comment:
Intentionally dropping version of unknown Linux OS. Gentoo becomes "linux24", which makes little sense. Better leave it at "linux" only.